### PR TITLE
Add container view to container apps 

### PR DESCRIPTION
**Pull request by motm32**
Part of #426 

Bringing this branch back from the dead 🧟

Here is how the view looks: 
image

At the moment we are only supporting displaying the information. In the future we hope to add support for adding/uploading environment variables. 

I also extended `RevisionDraftDescendantBase` instead of `ContainerAppsItem` so when we add further support it already basically has drafts/revisions support. Let me know if we want to change it to `ContainerAppsItem` instead.

### CODE REVIEW

**Comment by MicroFish91** [comment]
This is looking really nice 😎, can't wait to try it out later

**Comment by motm32** [comment]
This is how it looks in the case of multiple containers: 
image

File: src/tree/containers/EnvironmentVariableItem.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    _hideValue: boolean;
+    constructor(public readonly subscription: AzureSubscription,
+        public readonly containerApp: ContainerAppModel,
+        public readonly revision: Revision,
+        readonly containerId: string,
+        readonly container: Container,
+        readonly envVariable: EnvironmentVar) {
+        this._hideValue = true;
+    }
+    id: string = `${this.containerId}/environmentVariable`
```
Comment (MicroFish91) [inline src/tree/containers/EnvironmentVariableItem.ts:25]: The `id` here won't always be unique and could throw an error if you have multiple environment variables present.  Maybe adding on the `name` property at the end could alleviate this:

`${this.containerId}/environmentVariables/${this.envVariable.name}`

File: src/tree/containers/ContainersItem.ts [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+export class ContainersItem extends RevisionDraftDescendantBase {
+    id: string;
+    label: string;
+    private containers: Container[] = [];
+
+    constructor(public readonly subscription: AzureSubscription,
+        public readonly containerApp: ContainerAppModel,
+        public readonly revision: Revision,) {
+        super(subscription, containerApp, revision);
+        this.id = `${containerApp.id}/containers`;
```
Comment (MicroFish91) [inline src/tree/containers/ContainersItem.ts:31]: This id may not always be unique, for example in multiple revisions mode.

You can probably fix this by doing something similar to what I did in the ID for `ScaleRuleItem`!

**Pull request merged**